example/src/ThemeLoader.js

```diff
@@ -0,0 +1,70 @@
+import cssVarsPonyfill from 'css-vars-ponyfill';
+
+const THEME_URL_PARAM ='themeId';
+const THEME_CDN_URL   = './';
+
+class ThemeLoader {
+
+    static load() {
+
+        return new Promise(((resolve, reject) => {
+
+
+            /**
+             * Get the parameter `themeId` from the URL in case if it exists, and append
+             * it in a `link` element in the DOM
+             */
+            const applyThemeIfExists = () => {
+
+                const queryString = window.location.search.substr(1);
+
+                if (queryString === '') return;
+
+                queryString.split('&').forEach((item) => {
+
+                    const queryParam = item.split('=');
+
+                    if (queryParam[0] === THEME_URL_PARAM) {
+                        const styleElement= document.createElement('link');
+                        styleElement.type = 'text/css';
+                        styleElement.rel = 'stylesheet';
+                        styleElement.href = `${THEME_CDN_URL}${queryParam[1]}`;
```

Comment:
We also should detect the moment when this style is loaded.
See this: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#Stylesheet_load_events

Reply:
I think I got it. Pushed the changes